### PR TITLE
cleanup and fixed a broken code

### PR DESCRIPTION
**Pull request by mregandla**
1. Updated .gitignore
2. Removed unnecessary files from the repo
3. Fixed broken code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,9 +6,30 @@
...
 .example/lib/application_constants.dart
 
+# IntelliJ related
+*.iml
+*.ipr
+*.iws
+.idea/
+
+# Android related
+gradle.properties
```
Comment (mregandla) [inline .gitignore]: I think, gradle.properties should not be ignored. @phillwiggins Do you have any comments? Let me know if I need to fix this.

Comment (mregandla) [reply .gitignore]: Never mind, this is fixed now.

**Comment by phillwiggins** [comment]
Great work! All looks good.

File: .gitignore (hunk 1 above)
Comment (phillwiggins) [reply .gitignore]: Sorry for the delay, I've been away. Yes, you've done the correct thing by not adding gradle.properties to gitignote. Thanks

**Pull request merged**